### DISCUSSION TITLE
/api/<domain>/status returns "waiting" even though there are page views

### DISCUSSION THREAD

**Discussion by ellahin** [plausible/analytics#4143] [Self-Hosted Support]
Hi,

I just set up plausible CE.

The site is polling data as I can view page visits on the main page.

![image](https://github.com/plausible/analytics/assets/102624759/3661851a-2c48-4738-bec2-0f51f172fc5e)

However, when I click on the page to view data it says waiting for first page view and "/api/<domain>/status"  returns "waiting" as a response.

I have fully deleted all files and volumes to redeploy and I have the same result.

Kind regards,
Ella.

**Comment by ellahin** [comment]
Slight update, I've manually set stats_start_date in sites per #3725.
now the tiles load but there is no data.
![image](https://github.com/plausible/analytics/assets/102624759/4b7fc2c3-cdef-43ec-8e8c-e589f74425ec)

**Comment by ellahin** [comment]
The current visitor count does work 
![image](https://github.com/plausible/analytics/assets/102624759/b3c36591-ee83-46b7-84d1-2f7e5961f59d)

**Comment by ruslandoga** [comment]
👋 @ellahin 

Do you have a non-UTC timezone set in one of the containers? Do events start to show up after ~12 hours? Do you have events in ClickHouse, what are their timestamps? What are timestamps for the site in PostgreSQL?

**Reply by ellahin** [reply]
Views are starting to show up 12 hours later, the first original one doesn't though but all new ones do. I am using an external Postgres database that's +10 utc so that's most likely the issue.

**Reply by ruslandoga** [reply]
With that setup the events from the first 10 hours would probably never show up. All the events that arrived after 10 hours after website creation should show up just fine.

**Reply by ellahin** [reply]
Fair enough, will that only be the case from the first time the server is set up or every time a new domain is added?

**Reply by ruslandoga** [reply]
In v2.1.0 it would happen every time a new domain is added.

**Reply by ellahin** [reply]
If I changed the Postgres server to utc I'm assuming that should solve the issue? Would that affect any of the current logs?

**Reply by ruslandoga** [reply]
It would solve the issue for all sites created after that change. The current websites' stats_start_date and native_stats_start_at columns can be updated manually. Event data is stored in ClickHouse and wouldn't be affected.

**Reply by ellahin** [reply]
Makes sense, thanks for the help and advice really appreciate it!!
